src/Microsoft.ML.Transforms/Dracula/DraculaTransform.cs

```diff
@@ -0,0 +1,518 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.IO;
+using System.Linq;
+using Microsoft.ML;
+using Microsoft.ML.CommandLine;
+using Microsoft.ML.Data;
+using Microsoft.ML.Data.IO;
+using Microsoft.ML.EntryPoints;
+using Microsoft.ML.Internal.Utilities;
+using Microsoft.ML.Runtime;
+using Microsoft.ML.Transforms;
+
+[assembly: LoadableClass(DraculaTransformer.Summary, typeof(IDataTransform), typeof(DraculaEstimator), typeof(DraculaEstimator.Options), typeof(SignatureDataTransform),
+    DraculaTransformer.UserName, DraculaTransformer.LoaderSignature, "Dracula")]
+
+[assembly: LoadableClass(DraculaTransformer.Summary, typeof(DraculaTransformer), typeof(DraculaEstimator), null, typeof(SignatureLoadModel),
+    DraculaTransformer.UserName, DraculaTransformer.LoaderSignature)]
+
+[assembly: EntryPointModule(typeof(Dracula))]
+
+namespace Microsoft.ML.Transforms
+{
+    public class DraculaEstimator : IEstimator<DraculaTransformer>
```

Comment:
Moving @codemzs' [comment](https://github.com/dotnet/machinelearning/pull/4514#issuecomment-560502623) to a thread:

> @yaeldekel Shall we give it a better name as opposed to calling it "Dracula"? Seems a little informal to me.

Reply:
If `Dracula` is not used, `CountTargetEncoder` might be suitable.

Our old blog post lists DRACULA as "**D**istributed **R**obust **A**lgorithm for **C**ount-based **L**e**a**rning": https://blogs.technet.microsoft.com/machinelearning/2015/02/17/big-learning-made-easy-with-counts/

As an aside, there's also public slides: 
https://www.slideshare.net/SessionsEvents/misha-bilenko-principal-researcher-microsoft

Reply:
Good suggestion. Changing the name to `CountTargetEncoder`, unless someone has a different idea?